lib/src/test/java/uk/gov/di/ipv/core/library/helpers/AuthorizationRequestHelperTest.java

```diff
@@ -217,9 +221,10 @@ private void setupCredentialIssuerConfigMock() {
     }
 
     private void setupConfigurationServiceMock() {
-        when(configurationService.getCoreFrontCallbackUrl()).thenReturn(CORE_FRONT_CALLBACK_URL);
-        when(configurationService.getIpvTokenTtl()).thenReturn(IPV_TOKEN_TTL);
-        when(configurationService.getAudienceForClients()).thenReturn(IPV_ISSUER);
+        when(configurationService.getSsmParameter(ConfigurationVariable.CORE_FRONT_CALLBACK_URL))
+                .thenReturn(CORE_FRONT_CALLBACK_URL);
```

Comment:
Minor, but to prevent the naming clash/make it clearer here could rename the static class vars with `MOCK_` prefix or something? (ignore if this goes against convention 😄)

Reply:
Nice spot. Changed!